.ci/legacy_project/all_packages/android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     namespace = "com.example.all_packages"
-    compileSdkVersion flutter.compileSdkVersion
```

Comment:
Do we actually want to change this one? If we somehow broke it (maybe that's not even possible, but then... gradle), we would want to know presumably. And this is what the template generated at the time.

Reply:
> That said what would break is not our code but agp code

If that happened we would want to put in a Flutter Fix or an auto-migrator though, presumably? Getting a warning about needing to do that from our CI could be nice.

> I decided removing the last instance of compileSdkVersion was a good idea. If you feel strongly then I can revert this.

I would lean toward leaving it the way it was since in general the idea of `legacy_project` is to make absolutely minimal changes to it, but I'm fine with you making a call on it either way.

> I do not think gradle check runs against the legacy project so it wont cause a breakage.

Right, the check runs on individual packages, and won't include this file.

Reply:
reverted.